### PR TITLE
Update to + adding additional static tables 

### PR DESCRIPTION
**Pull request by catherine-2**
Brief comments on the purpose of your changes:

Hi! This PR includes:

1/ Adding 2 rows for Art Blocks collections that are launching next week to the existing metadata table 
2/ Adding similar new tables for other art platforms that use a single contract address for multiple collections 

I have made a system for myself to maintain these on an ongoing basis so as a heads up I may have small PRs every few weeks adding row(s) as needed. 

Let me know of any questions - thank you!

*For Dune Engine V2*
I've checked that:
General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [ ] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by soispoke** [comment]
Hi @catherine-2, looks like a parsing error is reported by the dbt slim ci github action, please check its details and see if all checks pass by clicking on `details` above 

Capture d’écran 2022-09-19 à 08 09 48

**Comment by catherine-2** [comment]
@soispoke  thank you! ah yep - just added the metadata tag to all tables. will make sure to do that going forward

**Comment by soispoke** [comment]
@catherine-2 Just a quick thought before I merge this: 
Did you consider standardizing column names across your different models ? It would be great to create a single table containing the different collection specific metadata views here, wdyt ?

On another note, these models still won't be exposed (still not added to the alter_table_properties table)

**Comment by catherine-2** [comment]
@soispoke  Yeah I tried to standardize them as much as possible. contract_address, collection_name and artist_name are used for all tables - the one exception is proof grails which has edition_name since they aren't collections

Let me spend some time thinking of a good way to combine them -- agree it would be super useful and the better long term solution. Currently you have to use different join logic with tokenId for each of these projects so I may need to add some new columns to make it usable (e.g. for some platforms they use ##,000,001 as the project id / token id format and some use ##,001 as the project id / token id format). 

Let's hold off merging while I work on this. I will come back to you on this PR with more detailed thoughts + a potential solution.

Also one question for you on table naming -- we've merged 2 tables already here and they have slightly different table formats in V2. Should I remove the "metadata_" at the start of the alias for the AB table? Or add it to the start of the ringers one? I don't have a strong view on which format should be used but think we want a consistent structure where it starts with nft_ethereum. or nft_ethereum_metadata.


**Art Blocks**
- Dune V2 table name: nft_ethereum_metadata.metadata_art_blocks_collections 
- Alias: metadata_art_blocks_collections
- Model name: nft_ethereum_metadata_art_blocks_collections

**Ringers**

- Dune V2 table name: nft_ethereum.metadata_ringers
- Alias: ringers
- Model name: nft_ethereum_metadata_ringers

**Comment by soispoke** [comment]
@catherine-2 awesome, lmk :) 
About your question, it should definitely the same as the ringers spells (remove metadata 👍 )

**Comment by catherine-2** [comment]
@soispoke  cool will fix that also - thank you!

**Comment by catherine-2** [comment]
@soispoke  I just finished updating all of the tables to be more consistent + added a combined table referencing the individual tables. Most of these changes vs. the last review are formatting (e.g. standardized column names) plus two new columns that make the joins to other tables possible. Please let me know of any questions or feedback!

Also re: your comment on not exposing yet - 👍 that sounds good. This PR covers a large portion of the data I am aggregating but isn't comprehensive yet. I'm working on figuring out how to add the remaining platforms (their data structure is a little different) and will open another PR once I have that together. Hopefully that one will be an easier review since we will be set on the overall table structure already!

File: models/nft/ethereum/metadata/nft_ethereum_metadata.yml
Comment (soispoke) [inline models/nft/ethereum/metadata/nft_ethereum_metadata.yml:72]: Could you rename this file as nft_ethereum_metadata_schema.yml (just for consistency)

Comment (soispoke) [reply models/nft/ethereum/metadata/nft_ethereum_metadata.yml:72]: It would also be nice to include the `models/nft/ethereum/metadata/nft_ethereum_metadata_art_platform_collections.sql` model in the schema .yml file

**Comment by catherine-2** [comment]
@soispoke  awesome thank you! Just updated the yml file name. And the nft_ethereum_metadata_art_platform_collections table is listed at the bottom of the yml file - let me know if anything looks off!

**Comment by soispoke** [comment]
oh you're right, missed it.

lgtm thanks again @catherine-2 !

**Review by soispoke — APPROVED** [review]

**Pull request merged**
